config/sections/mixins/batch.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+return [
+	'props' => [
+		/**
+		 * Activates the batch delete option for the section
+		 */
+		'batch' => function (bool $batch = false) {
+			return $batch;
+		},
+	],
+	'computed' => [
+		'batch' => function () {
+			return $this->layout === 'table' ? false : $this->batch;
```

Comment:
`Question:` Did you not allow this for table layout just because there is no selectable UI for the table layout yet? Or other reasons as well? 

If the former, I think the table layout is actually the one where people request most batch actions, so I would think we should then rather spend the time and implement selections for the table layout as well.

Reply:
Yes, the main reason is that I haven't found the time yet to implement it for table rows. It should not be too difficult though.

Reply:
I also just had a look at the table component and it shouldn't be that hard. As we could reuse the index column for the selection input.